charts/tigera-operator/templates/tigera-operator/02-serviceaccount-tigera-operator.yaml

```diff
@@ -1,5 +1,12 @@
+{{ $secrets := list }}
+{{ range $name := keys .Values.imagePullSecrets -}}
+{{ $item := dict "name" $name }}
+{{ $secrets = append $secrets $item }}
+{{ end }}
+
 apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: tigera-operator
   namespace: {{.Release.Namespace}}
+imagePullSecrets: {{ $secrets | toYaml | nindent 2 }}
```

Comment:
One minor thing, but this should be `{{- $secrets` to trim preceding spaces from this line